### PR TITLE
Add 4-15 weekly news post

### LINKED ISSUE #802: Weekly news blog post for 2016-04-15

### PR DESCRIPTION
**Pull request by ktbartholomew**
needs more tax puns. Fixes #802.

### CODE REVIEW

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/getcarina.com/build-0fbfce6c91/).

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/getcarina.com/build-3dedee4aa5/)

**Comment by ktbartholomew** [comment]
@kmsholcomb would you mind taking a look at this blog post? Thank you!

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/getcarina.com/build-f0e2f43a82/).

File: _posts/2016-04-15-weekly-news-container-day.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+In addition to the meetup, members of the Carina team will be present throughout the summit in the [“Rackspace Cantina”](http://blog.rackspace.com/relax-recharge-rackspace-cantina-openstack-summit-austin/). If you’re attending the summit, come see us on the rooftop patio at Michelada’s to relax and recharge!
+
+### Carina reboots completed
+
+Our team completed the process of rebooting all Carina instances on Tuesday, April 12th to patch a [Linux kernal bug](https://tech.vijayp.ca/linux-kernel-bug-delivers-corrupt-tcp-ip-data-to-mesos-kubernetes-docker-containers-4986f88f7a19) that delivered corrupt network data to containers. The reboots went smoothly, and all Carina clusters are up and running again. All containers that were configured to reboot automatically should have done so, but please reach out to us in [the forums](https://community.getcarina.com/) or in [#carina on freenode](https://botbot.me/freenode/carina/) if you’re seeing any issues with your clusters following the reboot.
+
+### Replay and slides of “Essential Docker Swarm”
```
Comment (everett-toews) [inline _posts/2016-04-15-weekly-news-container-day.md]: "Effective Docker Swarm"

File: _posts/2016-04-15-weekly-news-container-day.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+Docker has released [version 1.11 of Docker Engine and version 1.7 of Docker Compose](https://blog.docker.com/2016/04/docker-engine-1-11-runc/) with a lot of exciting changes under the hood and a few exciting user-facing changes, including DNS round-robin load balancing for simple service discovery. Coming soon to a cluster near you!
+
+### User feedback
+
+As a quick reminder, we live and thrive on user feedback. Bugs? Sharp edges? Want a pony? Please get in touch:
+
+* [Community (any and all feedback)](https://community.getcarina.com/)
+* [General feedback, bugs, etc.](https://github.com/getcarina/feedback)
+* [Website bugs](https://github.com/getcarina/getcarina.com/issues) (Also, request new tutorials here!)
+* [#carina on irc.freenode.net](https://botbot.me/freenode/carina/)
```
Comment (everett-toews) [inline _posts/2016-04-15-weekly-news-container-day.md]: `* Use #getcarina on [Twitter](https://twitter.com/)`

File: _posts/2016-04-15-weekly-news-container-day.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+In addition to the meetup, members of the Carina team will be present throughout the summit in the [“Rackspace Cantina”](http://blog.rackspace.com/relax-recharge-rackspace-cantina-openstack-summit-austin/). If you’re attending the summit, come see us on the rooftop patio at Michelada’s to relax and recharge!
+
+### Carina reboots completed
+
+Our team completed the process of rebooting all Carina instances on Tuesday, April 12th to patch a [Linux kernal bug](https://tech.vijayp.ca/linux-kernel-bug-delivers-corrupt-tcp-ip-data-to-mesos-kubernetes-docker-containers-4986f88f7a19) that delivered corrupt network data to containers. The reboots went smoothly, and all Carina clusters are up and running again. All containers that were configured to reboot automatically should have done so, but please reach out to us in [the forums](https://community.getcarina.com/) or in [#carina on freenode](https://botbot.me/freenode/carina/) if you’re seeing any issues with your clusters following the reboot.
+
+### Replay and slides of “Effective Docker Swarm”
+
+<img class="right" src="{% asset_path 2016-04-15-weekly-news/swarmnado.gif %}" style="max-width: 25%;" />
```
Comment (everett-toews) [inline _posts/2016-04-15-weekly-news-container-day.md]: Add image source caption?

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/getcarina.com/build-0cbc327025/).

File: _posts/2016-04-15-weekly-news-container-day.md (hunk 3 above)
Comment (ktbartholomew) [inline _posts/2016-04-15-weekly-news-container-day.md]: @everett-toews I yanked it from your slides. What _is_ the original source?

File: _posts/2016-04-15-weekly-news-container-day.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+In addition to the meetup, members of the Carina team will be present throughout the summit in the [“Rackspace Cantina”](http://blog.rackspace.com/relax-recharge-rackspace-cantina-openstack-summit-austin/). If you’re attending the summit, come see us on the rooftop patio at Michelada’s to relax and recharge!
+
+### Carina reboots completed
+
+Our team completed the process of rebooting all Carina instances on Tuesday, April 12th to patch a [Linux kernal bug](https://tech.vijayp.ca/linux-kernel-bug-delivers-corrupt-tcp-ip-data-to-mesos-kubernetes-docker-containers-4986f88f7a19) that delivered corrupt network data to containers. The reboots went smoothly, and all Carina clusters are up and running again. All containers that were configured to reboot automatically should have done so, but please reach out to us in [the forums](https://community.getcarina.com/) or in [#carina on freenode](https://botbot.me/freenode/carina/) if you’re seeing any issues with your clusters following the reboot.
```
Comment (everett-toews) [inline _posts/2016-04-15-weekly-news-container-day.md]: "reboot automatically" ==> `[restart automatically](https://docs.docker.com/engine/reference/commandline/run/#restart-policies-restart)`

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/getcarina.com/build-bf1aab75f6/)

File: _posts/2016-04-15-weekly-news-container-day.md (hunk 3 above)
Comment (everett-toews) [inline _posts/2016-04-15-weekly-news-container-day.md]: I found it on https://goto.docker.com/swarm-week.html

I do attribution [like this](https://github.com/getcarina/getcarina.com/blame/master/_tutorials/2016-03-01-overlay-networks.md#L25-L30). I think it suffices to say "Docker Inc."

**Comment by everett-toews** [comment]
My review is done. 

Good read!

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/getcarina.com/build-368d19c4da/).

File: _posts/2016-04-15-weekly-news-container-day.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
+---
+title: "Weekly news: Reboot Day, Meetup Day, Tax&nbsp;Day"
```
Comment (kmsholcomb) [inline _posts/2016-04-15-weekly-news-container-day.md]: "Weekly news: Reboot day, Meetup day, Tax day"

(More in keeping with the spirit of sentence-style capitalization, but emphasizing the occasions.)

File: _posts/2016-04-15-weekly-news-container-day.md [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+  - Swarm
+  - Carina
+  - Rackspace
+  - News
+authorIsRacker: true
+---
+
+{{ page.excerpt }}
+
+### Containers Meetup at OpenStack Summit Austin
```
Comment (kmsholcomb) [inline _posts/2016-04-15-weekly-news-container-day.md]: Containers meetup at ...

File: _posts/2016-04-15-weekly-news-container-day.md [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+
+The Carina team will be [hosting a containers meetup](https://getcarina.com/blog/container-day-austin-summit/) on April 26th, during the OpenStack Summit in Austin. The meetup will be a great time to see demos and discussions from early Carina users and other leaders in the containers ecosystem. The meetup is _not exclusive_ to summit attendees—anyone is welcome to come meet with our team and other Carina users.
```
Comment (kmsholcomb) [inline _posts/2016-04-15-weekly-news-container-day.md]: April 26
(no reason to add the "th" to the date; also, no need for the comma here)

... come and meet...

File: _posts/2016-04-15-weekly-news-container-day.md [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+
+In addition to the meetup, members of the Carina team will be present throughout the summit in the [“Rackspace Cantina”](http://blog.rackspace.com/relax-recharge-rackspace-cantina-openstack-summit-austin/). If you’re attending the summit, come see us on the rooftop patio at Michelada’s to relax and recharge!
```
Comment (kmsholcomb) [inline _posts/2016-04-15-weekly-news-container-day.md]: ...come and see us on the ... at Micheladas ...

File: _posts/2016-04-15-weekly-news-container-day.md [hunk 9]
```diff
@@ -0,0 +1,65 @@
...
+
+### Carina reboots completed
+
+Our team completed the process of rebooting all Carina instances on Tuesday, April 12th to patch a [Linux kernal bug](https://tech.vijayp.ca/linux-kernel-bug-delivers-corrupt-tcp-ip-data-to-mesos-kubernetes-docker-containers-4986f88f7a19) that delivered corrupt network data to containers. The reboots went smoothly, and all Carina clusters are up and running again. All containers that were configured to [restart automatically](https://docs.docker.com/engine/reference/commandline/run/#restart-policies-restart) should have done so, but please reach out to us in [the forums](https://community.getcarina.com/) or in [#carina on freenode](https://botbot.me/freenode/carina/) if you’re seeing any issues with your clusters following the reboot.
```
Comment (kmsholcomb) [inline _posts/2016-04-15-weekly-news-container-day.md]: ... Tuesday, April 12, to patch ... 

kernel

File: _posts/2016-04-15-weekly-news-container-day.md [hunk 10]
```diff
@@ -0,0 +1,65 @@
...
+### Use ObjectRocket’s MongoDB with Carina
+
+Databases are often a critical component for applications, but managing them yourself can be time-consuming and disaster-prone.
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">Databases will soon join mail servers on the list of things you don&#39;t ever want to host or manage yourself.</p>&mdash; Kelsey Hightower (@kelseyhightower) <a href="https://twitter.com/kelseyhightower/status/718291374798450688">April 8, 2016</a></blockquote>
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+Everett shows you how to heed [@kelseyhightower](https://twitter.com/kelseyhightower)’s advice and let ObjectRocket host MongoDB for you. [Using Carina with MongoDB from ObjectRocket](https://getcarina.com/docs/tutorials/data-stores-mongodb-prod/) gives you the best of both worlds: the speed and ease of development and deployment on Carina, plus the simplicity and reliability of ObjectRocket's managed MongoDB service.
+
+### Docker Releases Engine 1.11 and Compose 1.7
```
Comment (kmsholcomb) [inline _posts/2016-04-15-weekly-news-container-day.md]: Docker releases ...

**Comment by kmsholcomb** [comment]
Just a few minor suggestions, @ktbartholomew. :+1:

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/getcarina.com/build-368d19c4da/). :bow:

**Pull request merged**